src/api/auth.ts

```diff
@@ -346,13 +359,32 @@ export function changeUserRole(
 
 export function publishPackage(
   pkgName: string, username: string, organization: string,
-  version: string, auth: any): Promise<string> {
-    return new Promise((resolve, reject) => {
-      if (!organization) {
+  teamPermissions: [TeamPermissionData], version: string, auth: any): Promise<string> {
```

Comment:
you are declaring that Promise return string but then you return `auth` object, you can define here `any`.